kahuna/public/stylesheets/modules/_typography.scss

```diff
@@ -0,0 +1,14 @@
+/** =============================================
+
+_TYPOGRAPHY.SCSS - Type styles and sizes
+
+============================================= **/
+
+$baseFontFamily: "Guardian Agate Sans Web", Georgia;
```

Comment:
Georgie is a serif font. Better fallbacks for Agate would be
`"Lucida Sans Unicode", "Lucida Grande", sans-serif`
I think.